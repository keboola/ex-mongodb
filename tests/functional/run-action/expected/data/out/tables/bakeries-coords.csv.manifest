{"incremental":true,"columns":["w","n","zipcode","street","bakeries_id"],"column_metadata":{"w":[{"key":"KBC.datatype.nullable","value":true}],"n":[{"key":"KBC.datatype.nullable","value":true}],"zipcode":[{"key":"KBC.datatype.nullable","value":true}],"street":[{"key":"KBC.datatype.nullable","value":true}],"bakeries_id":[{"key":"KBC.datatype.nullable","value":true}]},"primary_key":["zipcode","street"]}