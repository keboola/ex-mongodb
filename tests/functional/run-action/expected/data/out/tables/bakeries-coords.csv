"w","n","zipcode","street","bakeries_id"
"-73.8747516","40.829474","10472","Westchester Avenue","5716054cee6e764c94fa9f31"
"-73.8767078","40.8290734","10472","Westchester Avenue","5716054cee6e764c94fad66f"
"-73.885417","40.82766","10459","Westchester Avenue","5716054bee6e764c94fa8c13"
"-73.8806669","40.8283447","10472","Westchester Avenue","5716054bee6e764c94fa841e"
"-73.8789604","40.8286012","10472","Westchester Avenue","5716054bee6e764c94fa8ff6"
"-73.8510158","40.8342588","10462","Westchester Avenue","5716054cee6e764c94faa105"
